code/modules/admin/admin.dm

```diff
@@ -2734,6 +2734,28 @@ var/global/noir = 0
 						logTheThing("admin", usr, null, "used Farty Party secret")
 						logTheThing("diary", usr, null, "used Farty Party secret", "admin")
 
+					if("give-legs")
+						if(src.level >= LEVEL_ADMIN)
+							var/list/atoms_to_leg = list()
+							var/pathname = input("Path of the things you want to leg like (/obj/item/paper_bin)","path:", null) as null|text
+							if(!pathname)
+								return
+							if (alert("Do you really wanna give things legs?", "LEG LEG", "Sure thing!", "Not really.") == "Sure thing!")
+								for(var/atom/movable/A in world) //Build the god forsaken list
+									if(istype(A, text2path(pathname)))
+										atoms_to_leg += A
+								if(atoms_to_leg.len >= 1)
+									logTheThing("admin", usr, null, "used give legs secret")
+									logTheThing("diary", usr, null, "used give legs secret", "admin")
```

Comment:
In discord you mentioned renaming it to be clear that it was a mass change, I think you maybe forgot the log text though.

```suggestion
									logTheThing("admin", usr, null, "used Mass give things legs secret")
									logTheThing("diary", usr, null, "used Mass give things legs secret", "admin")
```

Reply:
oh thanks for catching that